securedrop_client/gui/widgets.py

```diff
@@ -1799,6 +1799,10 @@ def _on_export_clicked(self):
         """
         Called when the export button is clicked.
         """
+        if not self.controller.downloaded_file_exists(self.file.uuid):
+            self.controller.sync_api()
```

Comment:
do we need to resync the API? we could just update the file with `is_downloaded=False` and then update the UI. I'll file a followup for this though since imho we'll should do this via sending a signal to `FileWidget` which is a larger change